### DISCUSSION TITLE
Copy profile information feature doesn't work?

### DISCUSSION THREAD

**Discussion by leryeo** [dayanch96/BHTikTok-Plus#38] [General]
![IMG_1361](https://github.com/dayanch96/BHTikTok-Plus/assets/156037577/398e4fe9-c189-44d7-ae12-4d242b1deb96)

As you can see in this example whenever you press or long press the profile information it brings up the translation interface. I went into language settings and disabled translation but the translation interface still pops up when I go to long press the profile information? In this picture I am trying to copy the profile information that includes the email address rather than having to type it all out.

**Comment by dayanch96** [comment]
There is no need to darn identical discussions. I already answered. The latest version in the release is 31.5.0, I am not responsible for newer versions of TikTok and do not plan to release an update. Thank you
